go.mod

```diff
@@ -1,8 +1,26 @@
 module github.com/rocketlaunchr/google-search
 
-go 1.12
+go 1.20
```

Comment:
Is go 1.20 strictly necessary? Perhaps 1.16 might be more suitable.

Reply:
Restored in dfdb3593e1e2388020736c362ba2bd21d4b6222e